### PR TITLE
MODFEE-151 correctly specify permissions

### PR DESCRIPTION
**Pull request by zburke**
Some `modulePermissions` for the path `DELETE /accounts/{id}` were
incorrectly listed under `permissionsRequired` instead.

Fixes [MODFEE-151](https://issues.folio.org/browse/MODFEE-151)

### CODE REVIEW

File: NEWS.md
Comment (marcjohnson-kint) [inline NEWS.md:4]:
```suggestion
* Grants permission for messages to be published to pub-sub when deleting an account (MODFEE-151)
```

**Review by marcjohnson-kint — APPROVED** [review]

**Comment by zburke** [comment]
Bump, @alexanderkurash. It would be really nice to have this included in the Iris release so we can more easily clean up the data in bugfest without having to manually hack at permissions.

**Comment by marcjohnson-kint** [comment]
> Bump, @alexanderkurash. It would be really nice to have this included in the Iris release so we can more easily clean up the data in bugfest without having to manually hack at permissions.

@zburke Are you waiting for an approval from @alexanderkurash to merge this change?

**Comment by zburke** [comment]
Yes I am. I am not a regular contributor to this module so I was aiming for two approvals from contributors or one from the dev lead. But if you think this is good to merge, I'm happy to do so, or have you do so.

**Comment by marcjohnson-kint** [comment]
> Yes I am. I am not a regular contributor to this module so I was aiming for two approvals from contributors or one from the dev lead. But if you think this is good to merge, I'm happy to do so, or have you do so.

I've merged it. If I followed your change correctly, the descriptor used to state that the client needed publishing permissions to delete an account whereas now it does not and the endpoint has them itself.

**Pull request merged**
